xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -268,8 +337,26 @@ private String exportXAR(XWikiContext context) throws XWikiException, IOExceptio
         String licence = request.get("licence");
         String version = request.get("version");
         String name = request.get("name");
-        String[] pages = request.getParameterValues("pages");
-        boolean all = ArrayUtils.isEmpty(pages);
+        String[] checkedPages = request.getParameterValues("pages");
+        String[] uncheckedPages = null;
+        String checkedPagesRequestValue = request.get("checked-pages");
+        String uncheckedPagesRequestValue = request.get("unchecked-pages");
+        String otherPagesRequestValue = request.get("other-pages");
+
+        boolean all = ArrayUtils.isEmpty(checkedPages) && StringUtils.isEmpty(otherPagesRequestValue) && StringUtils.isEmpty(checkedPagesRequestValue);
```

Comment:
This line is exceeding [120 characters](https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/main/resources/checkstyle.xml#L214).